src/ECER.Clients.RegistryPortal/ECER.Clients.RegistryPortal.Server/Communications/CommunicationsEndpoints.cs

```diff
@@ -24,6 +24,7 @@ public void Register(IEndpointRouteBuilder endpointRouteBuilder)
       var query = new UserCommunicationQuery
       {
         ByRegistrantId = userContext!.UserId,
+        ByParentId = parentId,
```

Comment:
Do we need to add in a check that parentId is a valid guid? 

Like what we did for draftapplications endpoint 
bool IdIsNotGuid = !Guid.TryParse(id, out _); if (IdIsNotGuid && id != null) { id = null; }

Reply:
This is a nullable field, we cannot validate it in the endpoint because its value could be null there, however, I added a validation logic in the repository.